source/scale/opensearch-setup.rst

```diff
@@ -0,0 +1,61 @@
+  c. (Optional) Enter **Server Password** associated with the username. For OpenSearch, leave this field blank.
+  d. Set **Enable Cluster Sniffing** (Optional). Sniffing finds and connects to all data nodes in your cluster automatically. For OpenSearch, this field should be set to ``false``.
+
+.. tip::
+
+  Optional CA and client certificate configuration settings are available for use with basic auth credentials or to replace them. See the :ref:`Elasticsearch configuration settings <configure/environment-configuration-settings:elasticsearch>` documentation for details.
+
+5. Select **Test Connection** and then select **Save**. If the server connection is unsuccessful you won't be able to save the configuration or enable searching with OpenSearch.
+
+6. Select **Build Index** to build the post index of existing posts. This process can take up to a few hours depending on the size of the post database and number of messages. The progress percentage can be seen as the index is created. To avoid downtime, set **Enable Elasticsearch for search queries** to ``false`` so that database search is available during the indexing process.
+
+  .. important::
+
+    Complete bulk indexing before enabling OpenSearch in the next step. Otherwise, search results will be incomplete.
+
+7. Enable Elasticsearch by setting **Enable Elasticsearch for search queries** to ``true``, and setting **Enable Elasticsearch for autocomplete** to ``true``. 
+
+8. Save your configuration updates and restart the Mattermost server.
+
+.. note::
+
+  - Additional advanced Elasticsearch settings for large deployments can be configured outside the System Console in the ``config.json`` file. Read the :ref:`Elasticsearch configuration settings <configure/environment-configuration-settings:elasticsearch>` documentation to learn more.
+  - If your deployment has a large number of posts (typically in excess of one million but not strictly defined), the reindexing progress percentage may stay at 99% for a long time. The size of the data to be indexed is estimated, and on large databases, estimations can become inaccurate. While progress estimates may be inaccurate, and the progress percentage may appear stuck at near completion, indexing will continue behind the scenes until complete.
+  - Search results for files shared before upgrading to Mattermost Server v5.35 may be incomplete until an extraction command is run using the :ref:`mmctl <manage/mmctl-command-line-tool:mmctl extract>`. After running this command, the search index must be rebuilt. Go to **System Console > Environment > Elasticsearch > Bulk Indexing**, then select **Index Now** to rebuild the search index to include older file contents.
+  - For high post volume deployments, we strongly encourage you to read and properly configure the Mattermost :ref:`LiveIndexingBatchSize <configure/environment-configuration-settings:live indexing batch size>` configuration setting.
+    
+Limitations
+------------
+
+TBD
```

Comment:
Should we include a TBD or just delete the entire section? May be a placeholder for SME input?

Reply:
Yes - this is a short-term placeholder for SME input (assuming there are OS limitations to note)